### ISSUE TITLE
Make LONG_POLLING_INTERVAL configurable

### ISSUE THREAD

**Issue by jhorman** [spulec/PyQS#60]
I would like to be able to have workers shutdown more quickly. Having this set to 20 when the queue is empty means slow deploys.

**Comment by spulec** [comment]
Agreed. A PR would be very welcome. Setting this as a feature request.

**Comment by Tenzer** [comment]
Would it not be more ideal if the polling would be interrupted when a SIGTERM signal is received? That way you would get the benefit of having the optimal long polling (with fewest possible requests) along with quick shutdowns.

I don't know how difficult that would be to achieve though.
